examples/SharedMemory/PhysicsServerCommandProcessor.cpp

```diff
@@ -2775,7 +2797,9 @@ void PhysicsServerCommandProcessor::deleteDynamicsWorld()
 		for (i = m_data->m_dynamicsWorld->getSoftBodyArray().size() - 1; i >= 0; i--)
 		{
 			btSoftBody* sb = m_data->m_dynamicsWorld->getSoftBodyArray()[i];
+#ifdef SKIP_DEFORMABLE_BODY
```

Comment:
is this intended? Only when SKIP_DEFORMABLE_BODY is skipped the sb is removed?

Reply:
No. I've fixed it.